content/community/examples.md

```diff
@@ -21,3 +21,4 @@ There are many example projects created by the React community. Feel free to add
 * **[Bitcoin Price Index](https://github.com/mrkjlchvz/bitcoin-price-index)** Simple bitcoin price index data from CoinDesk API.
 * **[Builder Book](https://github.com/builderbook/builderbook)** Open source web app to write and host documentation or sell books. Built with React, Material-UI, Next, Express, Mongoose, MongoDB.
 * **[GFonts Space](https://github.com/pankajladhar/GFontsSpace)** A space which allows user to play with Google fonts. Built with React, Redux and React-Router.
+* **[React Most Wanted](https://github.com/TarikHuber/react-most-wanted)** A react starter kit using Create React App (& NOT ejecting!), Material-UI, redux and Firebase to create a new react project with all "Most Wanted" features.
```

Comment:
Would it be ok with you to use "and" instead of "&"? I feel like the usage of the ampersand here is not justified.
```suggestion
* **[React Most Wanted](https://github.com/TarikHuber/react-most-wanted)** A react starter kit using Create React App (and NOT ejecting!), Material-UI, redux and Firebase to create a new react project with all "Most Wanted" features.
```

Reply:
Sure :) No idea why I wrote it like that.